openpype/modules/deadline/plugins/publish/submit_fusion_deadline.py

```diff
@@ -220,12 +221,29 @@ def process(self, instance):
             # these paths, such as if slaves are running Linux and the
             # submitter is on Windows.
             "PYTHONPATH",
```

Comment:
Sorry about this - but just saw this one here now too. Pretty sure local `PYTHONPATH` should not be submitted along either.